### PR TITLE
NODE-1106 changes to update OpenAI node and remove OpenAI Assistant node

### PR DESCRIPTION
**Pull request by freakwriter**
Followed instructions in: https://linear.app/n8n/issue/NODE-1106/docs-for-ai-node-tweaks-and-openai-node-update, tried to treat each item as a separate commit for quick review.

Two explicit review points:
- There was no list item to update the nav in mkdocs.yml to have the new openai URL instead (integrations/builtin/app-nodes/n8n-nodes-langchain.openai rather than integrations/builtin/app-nodes/n8n-nodes-base.openai.md). I'm assuming the redirect will handle that and that change would come at a future date; I just wanted to note this in case it was a change I should have made.
- Did I format the deprecation warning about OpenAI Assistant properly?

### CODE REVIEW

File: docs/integrations/builtin/app-nodes/n8n-nodes-langchain.openai.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Use the OpenAI node to automate work in OpenAI, and integrate OpenAI with other
 
 On this page, you'll find a list of operations the OpenAI node supports and links to more resources.
 
+/// note | OpenAI assistant node
```
Comment (StarfallProjects) [inline docs/integrations/builtin/app-nodes/n8n-nodes-langchain.openai.md:13]:
```suggestion
/// note | OpenAI Assistant node
```

File: docs/integrations/builtin/app-nodes/n8n-nodes-langchain.openai.md [hunk 2]
```diff
@@ -48,3 +52,21 @@ Refer to [OpenAI's documentation](https://beta.openai.com/docs/introduction){:ta
...
+Once you add a tool connection, the OpenAI node becomes a root node, allowing it to form a cluster node with the tools sub-nodes. See [Node types](/integrations/builtin/node-types/#cluster-nodes) for more information on cluster nodes and root nodes.
+
+### Operations that support tool connectors
+
+* Assistant
+	* Message Assistant
+* Text
+	* Message Model
+
+### Related resources
```
Comment (StarfallProjects) [inline docs/integrations/builtin/app-nodes/n8n-nodes-langchain.openai.md:70]: Suggest only having one 'Related resources' section. Please move this into the main one, and rename the link (so we have one general link to the OpenAI API docs, and one specifically for assistants)

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
See comments on the files

**Comment by StarfallProjects** [comment]
@freakwriter good catch with the nav in mkdocs.yml: please do go ahead and update this.

**Comment by freakwriter** [comment]
@StarfallProjects I've incorporated your feedback for n8n-nodes-langchain.openai.md and updated the nav in mkdocs.yml.

**Review by StarfallProjects — APPROVED** [review]

**Pull request merged**